templates/csi.yaml.j2

```diff
@@ -328,6 +404,21 @@ spec:
           volumeMounts:
             - name: varlog
               mountPath: "/var/log/gluster"
+        - name: csi-resizer
+          image: k8s.gcr.io/sig-storage/csi-resizer:v1.0.1
```

Comment:
can you use `docker.io/raspbernetes/csi-external-resizer:1.0.0` here instead of above? That way, we don't break arm64 and arm/v7 support.

Reply:
done!